cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -470,10 +505,12 @@ def _get_ssh_client(self, raise_on_error=True, timeout=5000):
                 username=self.ssh_username,
                 password=self._get_password(),
                 ssh_key=self._get_ssh_key(),
+                host_key=self.host_key if not self.skip_host_key else None,
```

Comment:
Question, maybe 
```
self.host_key if self.host_key and not self.skip_host_key else None
```

because if `self.host_key` is not set it will return False.